tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -399,28 +412,28 @@ def set_geo_rep_session(self, cluster_data):
                                 "for volume" + str(ex)
                             }
                         )
-            cluster.details["geo_rep"] = {}
-            cluster.details["geo_rep"]["total"] = geo_rep_mapper["total"]
-            cluster.details["geo_rep"]["up"] = geo_rep_mapper["up"]
-            cluster.details["geo_rep"]["down"] = geo_rep_mapper["down"]
-            cluster.details["geo_rep"]["partial"] = geo_rep_mapper["partial"]
-            cluster.details["geo_rep"]["created"] = geo_rep_mapper["created"]
-            cluster.details["geo_rep"]["stopped"] = geo_rep_mapper["stopped"]
-            cluster.details["geo_rep"]["paused"] = geo_rep_mapper["paused"]
+            cluster["geo_rep"] = {}
+            cluster["geo_rep"]["total"] = geo_rep_mapper["total"]
```

Comment:
Better form a dict first then set in cluster['geo_rep']

Reply:
i will change it